myokit/formats/sbml/__init__.py

```diff
@@ -33,5 +34,18 @@ def importers():
     """
     return dict(_importers)
 
```

Comment:
Can we get a 

```
from ._writer import (  # noqa
    write_file,
    write_string,
    SBMLWriter,
)
```

here? (With updated imports in other files, and mention of these 3 in the docs, sorry!)

Reply:
sure, done